numba_dppy/compiler.py

```diff
@@ -455,7 +456,7 @@ def _pack_argument(self, ty, val, sycl_queue, device_arr, access_type):
             # we get the date back to host if have created a
             # device_array or if access_type of this device_array
             # is not of type read_only and read_write
-            usm_buf, usm_ndarr, orig_ndarray = device_arr
+            usm_ndarr, orig_ndarray = device_arr
             np.copyto(orig_ndarray, usm_ndarr)
```

Comment:
To be consistent with the `orig_ndarray`, let us do s/usm_ndarr/usm_array. I do not want to call it `usm_ndarray` as that can be confusing given `usm_ndarray` is an actual data type in dpctl.